tokens/ply.json

```
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0x7cC183Af8d90015e5AA935A5240bE1c2b367B877",
+    "aurora_address": "0x3d4205511c25ccd29e3fff6fba0d321b44bb7980",
+    "name": "Aurigami Token",
```

This should match ERC-20 metadata. 

```suggestion
    "name": "PLY",
```

---

We have just deployed another ERC20 token on Ethereum with the name "Aurigami Token"